packages/app-store/cal-ai/DESCRIPTION.md

```diff
@@ -31,4 +31,6 @@ items:
 
 
 ### Or, doing multiple things:
-- Cancel all my meetings tomorrow then book a quick chat with @joe at noon, a long meeting with @jane at 1 (both tomorrow). Also see if Toby is down to grab dinner this week (toby@gmail.com). Finally send me my updated agenda of everything I have coming up in the next couple days.
\ No newline at end of file
+- Cancel all my meetings tomorrow then book a quick chat with @joe at noon, a long meeting with @jane at 1 (both tomorrow). Also see if Toby is down to grab dinner this week (toby@gmail.com). Finally send me my updated agenda of everything I have coming up in the next couple days.
+You will need a [Deepgram](https://console.deepgram.com/) API Key to transcribe calls in the app.
+Go to `Settings -> Admin -> Apps interface` and add the `deepgram_api_key` for `Cal.ai`.
```

Comment:
does every user need a Deepgram API Key? Can we not provide one for our users?

Reply:
You could, but then you probably need one of their plans depending on the usage?